problem3.py

```diff
@@ -0,0 +1,20 @@
+'''
+// Time Complexity : O(m + n)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Yes
+// Three line explanation of solution in plain english : same implementation as in class
```

Comment:
Kindly do not mention as "same implementation as in class". The intension for asking you to describe the approach in 3 lines would give you a practice to communicate the approach in a concise manner. Also, the TA's checking your solution might not know what was discussed in the class. We would want to understand the thought process behind your approach.